### PR TITLE
[WIP] Improve Fastlane pipeline

### PR DESCRIPTION
**Pull request by orta**
- [ ] Betas on TestFlight
- [x] Key Validation
- [ ] Use `gym` for deploys
- [ ] Submit via fastlane

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -8,7 +8,6 @@ use_frameworks!
 inhibit_all_warnings!
 
 plugin 'cocoapods-keys', {
-    :project => "Folio",
```
Comment (orta) [inline Podfile]: @sarahscott this should require you to delete the corresponding file in `~/.cocoapods/keys/` - ping me when you merge this

**Comment by orta** [comment]
@krausefx - I've been looking at this in Snapshots

``` ruby
setup_for_device_change do |device, udid, language|
  puts "Running #{language} on #{device}"
end
```

Because  I want to put in my own stubbed sqlite db into the app's user folder. Is there a way to get a hook up _after_ the app may have been built first? So that it will exist on the filesystem? Or should I try use it in app code for example?

**Comment by KrauseFx** [comment]
@orta Don't start integrating `snapshot`, I'm working on a new version to get rid of UI Automation and replace it with UI Tests.

**Comment by orta** [comment]
All of this work has been done in later PRs.

**Pull request closed without merging**